### ISSUE TITLE
Please upgrade joi

### ISSUE THREAD

**Issue by Vanuan** [jeffbski/wait-on#37]
```
yarn add wait-on@3.2.0
[1/4] Resolving packages...
warning wait-on > joi > hoek@5.0.4: This version is no longer maintained. Please upgrade to the latest version.
```

**Comment by paulmelnikow** [comment]
This was done in #38 and published as 3.3.0. 🚀

**Comment by jeffbski** [comment]
Check out the latest RC, I updated all the dependencies.

I have a fresh rewrite of the wait-on implementation which I think is simpler and hopefully will remove some of the race conditions that were possible.

You can try out the new version 4.0.0-rc.0 by

npm i wait-on@next
Let me know if this fixes the issues you were having.

Thanks!
